docs/dashboards/subscriptions.md

```diff
@@ -58,29 +58,33 @@ If there are no results, you can tell Metabase to skip sending the email.
 
 Here you can set filters to apply when Metabase sends the subscription.
 
-### Attach results
+## Attach results
 
-Tell Metabase if it should attach results to the email as a file, in addition to displaying the table in the email body.
+When configuring a subscription, you can tell Metabase if it should attach results to the email as a file, in addition to displaying the table in the email body.
 
 If you've added filters to your dashboard and set default values for those filters, Metabase will apply those default values to your subscriptions, filtering the results of all questions that are connected to those filters when the subscriptions are sent. To learn more, check out [dashboard filters](./filters.md).
 
-#### File format
+### File format
 
 You can choose between the following file formats:
 
 - .csv
 - .xlsx
 - .json
 
-#### Use unformatted values in attachments
+### Use unformatted values in attachments
 
 If you check this box, Metabase will drop any visualization settings applied to the questions' results (e.g., date formatting) and send only the raw, unformatted results.
 
-#### Questions to attach
+### Questions to attach
 
-Here you can specify which questions Metabase should attach results for.
+Here you can specify which questions Metabase should attach results for. If you don't select any questions, Metabase will not sent any attachments.
```

Comment:
Adding this sentence because I forget about ti literally every time i configure a subscription